src/Analysis/GWASV2/Components/DismissibleMessage/DismissibleMessage.jsx

```diff
@@ -7,8 +7,13 @@ const DismissibleMessage = ({
   title = 'Placeholder Title',
   description = 'placeholder description',
```

Comment:
I'm wondering if we want the user to be able to always be able to dismiss the messages. We might want to add a prop to the component so the user cannot dismiss the error message manually. Do you think there will be error messages we do not want the user to be able to dismiss (i.e. some critical error?)

Reply:
I thought about this scenario...We have only one such moment at the end of the flow, where we should not allow the user to trigger a workflow for a population of size == 0. I'll work on something for that...not sure yet what would be the best approach.

Reply:
Remaining population of size==0 will cause HARE dropdown to be empty, which will disable the "Next" button. I think that for now this is a good solution, but a better/clearer validation could be added in the future. This current one might also have some cases where it fails to disable the "Next" button, as I have not tested all scenarios:

<img width="719" alt="Screenshot 2023-01-26 at 18 25 36" src="https://user-images.githubusercontent.com/2900303/214906123-20baefad-cbf3-4b97-be50-740f6a293d29.png">